### PR TITLE
add last tagged sort for saved variants

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: ui/shared/utils/constants.js [hunk 1]
```diff
@@ -809,6 +810,15 @@ const VARIANT_SORT_OPTONS = [
...
         (acc, geneId) => (genesById[geneId] ? acc + genesById[geneId].omimPhenotypes.length : acc), 0),
   },
+  {
+    value: SORT_BY_TAGGED_DATE,
+    text: 'Last Tagged',
+    comparator: (a, b, genesById, user, tagsByGuid) =>
+      (b.tagGuids.map(
+        tagGuid => (tagsByGuid[tagGuid] || {}).lastModifiedDate).sort()[b.tagGuids.length - 1] || '').localeCompare(
+        a.tagGuids.map(tagGuid => (tagsByGuid[tagGuid] || {}).lastModifiedDate).sort()[a.tagGuids.length - 1] || '',
+      ),
```
Comment (ShifaSZ) [inline ui/shared/utils/constants.js:820]: The last parenthesis matches the one of `localeCompare(` in line 818, not the one in line 817. Rearrange the alignment of parentheses in these lines will make them more readable.

**Review by ShifaSZ — CHANGES_REQUESTED** [review]
There is one comment.

**Comment by hanars** [comment]
passing tests: https://travis-ci.org/github/broadinstitute/seqr/builds/756601068

File: ui/shared/utils/constants.js (hunk 1 above)
Comment (hanars) [reply ui/shared/utils/constants.js:820]: The indentation is correct, if I change it  I get a JS linting error so I am leaving this as is

**Pull request merged**